tests/FakeItEasy.Analyzer.Tests.Shared/Helpers/CodeFixVerifier.Helper.cs

```diff
@@ -0,0 +1,84 @@
+﻿namespace FakeItEasy.Analyzer.Tests.Helpers
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading;
+    using Microsoft.CodeAnalysis;
+    using Microsoft.CodeAnalysis.CodeActions;
+    using Microsoft.CodeAnalysis.Formatting;
+    using Microsoft.CodeAnalysis.Simplification;
+
+    /// <summary>
+    /// Diagnostic Producer class with extra methods dealing with applying codefixes
+    /// All methods are static
+    /// </summary>
+    public abstract partial class CodeFixVerifier
+    {
+        /// <summary>
+        /// Apply the inputted CodeAction to the inputted document.

```

Comment:
I did not know that "inputted" was an English word. The things one learns!
It hits me like a punch to the eye.
I'm not suggesting a change.